src/v3/src/components/Form/renderers.tsx

```diff
@@ -96,7 +96,14 @@ export default [
     renderer: WebAuthNSubmitButton,
   },
   {
-    tester: ({ name }: FieldElement) => name?.endsWith('phoneNumber'),
+    tester: (uischema: FieldElement) => {
+      const {
+        options: {
+          inputMeta: { name } = {},
+        } = {},
+      } = uischema;
+      return name?.endsWith('phoneNumber');
+    },
```

Comment:
What prompted using the name from `inputMeta` vs the top level name prop?

Reply:
remove duplicated values